Controllers/OrderController.js

```diff
@@ -41,4 +41,20 @@ let createNewOrder = (req, res) => {
     })
 }
 
-module.exports.createNewOrder = createNewOrder
\ No newline at end of file
+const addToOrder = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        try {
+            await OrderService.addOneItemToOrder(db, req)
```

Comment:
Suggest changing addOneItemToOrder so it gets given the individual item rather than the whole request object. Also worth doing some validation against this in the same way as your add order (using ajv)